core/runtime.rs

```diff
@@ -3216,6 +3216,90 @@ assertEquals(1, notify_return_value);
     runtime.run_event_loop(false).await.unwrap();
   }
 
+  #[tokio::test]
+  async fn test_sync_op_serialize_object_with_numbers_as_keys() {
```

Comment:
Could these tests be done in `serde_v8` crate instead of `core`?

Reply:
@bartlomieju I think it can be done but if we use this test, the way it is written currently, it would use things like `Extension` and `JSRuntime` that aren't present in that file. This test also runs inline JS scripts that use the `Deno` global for the native ops that are passed via the `core` property and it looks like there are no existing tests in `serde_v8` that does anything like this.

Reply:
@bartlomieju added a deserialization test to `serve_v8` that essentially tests the same scenario but doesn't use the `Deno.core.op...` API, PTAL!